presto/presto.sh

```diff
@@ -29,8 +29,8 @@ else
   readonly CONNECTOR_JAR="$(find /usr/lib/hadoop/lib -name 'gcs-connector-*.jar')"
 fi
 readonly PRESTO_BASE_URL=https://repo1.maven.org/maven2/com/facebook/presto
-readonly PRESTO_VERSION='0.206'
-readonly HTTP_PORT='8080'
+readonly PRESTO_VERSION='0.221'
+readonly HTTP_PORT="$(/usr/share/google/get_metadata_value attributes/dataproc-presto-port || 8080)"
```

Comment:
This can't work, but I don't know the proper way of using attributes. @medb ?

Reply:
We need to replace `8080` with `echo 8080` to make it work

Reply:
@medb is `/usr/share/google/get_metadata_value attributes/dataproc-presto-port` an executable or a plain text file?
If the latter, we need:
```
...=$(cat /usr/share/google/get_metadata_value attributes/dataproc-presto-port || echo 8080)
```

BTW Does Dataproc come woth some utility command for getting init param value (or default) in some easier way?

Reply:
`/usr/share/google/get_metadata_value` is an executable and `attributes/dataproc-presto-port` its parameter - this is client for [GCE metadata service](https://cloud.google.com/compute/docs/storing-retrieving-metadata).

No, we intentionally do not provide utility bash functions to not create backward-compatibility problems, we usually define such functions in each init action when necessary.

Reply:
sorry, i misread that as/a/very/long/path/to/a/file

Reply:
Ack.